appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/eia-development-description/eia-development-description.mapper.js

```diff
@@ -1,13 +1,31 @@
 import { appealShortReference } from '#lib/appeals-formatter.js';
 import { radiosInput } from '#lib/mappers/index.js';
 import { APPEAL_EIA_DEVELOPMENT_DESCRIPTION } from 'pins-data-model';
-import { snakeCaseToSpaceSeparated, capitalizeFirstLetter } from '#lib/string-utilities.js';
 
 /**
  * @typedef {import('../../appeal-details.types.js').WebAppeal} Appeal
  * @typedef {import('../lpa-questionnaire.service.js').LpaQuestionnaire} LpaQuestionnaire
  */
 
+export const eiaDescriptionDict = {
```

Comment:
This is a nice idea, it would be good to have these label constants for every row in the service eventually (too big for this ticket obviously). Just a pedantic point, I don't think the `Dict` part of the name is necessary - it could be removed or changed to the fully-qualified `Dictionary`.

Reply:
changed to `eiaDescriptions`